tools/tools.yml

```diff
@@ -209,15 +209,74 @@
       site: https://github.com/andrea-cuttone/geoplotlib/wiki/User-Guide
       badges: pypi
 
-- name: Other domain-specific
-  intro: Tools focused on specific research or application areas other than those above.
+- name: Graphs and networks
+  intro: Tools to visualize graphs (networks).
   packages:
 
     - repo: networkx/networkx
       site: https://networkx.github.io
       appveyor_project: dschult/networkx-pqott
       badges: travis, appveyor, codecov, pypi, conda, site
 
+    - repo: skewed/graph-tool
+      # repo: https://git.skewed.de/count0/graph-tool
+      site: http://graph-tool.skewed.de
+      conda_channel: conda-forge
+      badges: conda, site
+
+    - repo: igraph/python-igraph
+      site: https://igraph.org/python
+      conda_channel: conda-forge
+      appveyor_project: ntamas/python-igraph
+      badges: travis, appveyor, pypi, conda, site
+
+    - repo: xflr6/graphviz
+      site: https://graphviz.readthedocs.io
+      conda_channel: conda-forge
+      badges: travis, codecov, pypi, conda, rtd, site
```

Comment:
You might need to specify the conda and/or pip package names explicitly in some cases, including this one? 

E.g. conda-forge/graphviz is the underlying graphviz library (without any python interface), while https://pypi.org/project/graphviz/ is https://github.com/xflr6/graphviz's python bindings (for which graphviz is required). The equivalent conda-forge package is conda-forge/python-graphviz. Meanwhile https://pypi.org/project/graphviz-python/ is an abandoned/empty project (the underlying graphviz library included python bindings at some point...maybe it still does?). Hope I typed that out right - it's confusing! 

(There are examples of other tools in this file that have explicit package names.)

Reply:
Thanks! I didn't know we could specify package names.